vmdb/tools/fix_auth/models.rb

```diff
@@ -17,8 +17,18 @@ class FixMiqDatabase < ActiveRecord::Base
     self.table_name = "miq_databases"
     self.password_columns = %w(registration_http_proxy_server session_secret_token csrf_secret_token)
 
-    def self.hardcode(old_value, _new_value)
-      super(old_value, SecureRandom.hex(64))
```

Comment:
I can't find what `super` is doing.  Link to this method?  I only see this existing implementation.  
`vmdb/tools/fix_auth/models.rb:    def self.hardcode(old_value, _new_value)`